Scripts/Proxies/ProxyAnimator.cs

```diff
@@ -27,26 +30,36 @@ public void Apply(Transform transform)
     [SerializeField]
     ActionMap m_ProxyActionMap;
 
-    ProxyHelper.ButtonObject[] m_Buttons;
+    [SerializeField]
+    ProxyAffordanceMap m_AffordanceMap;
+
+    Affordance[] m_Affordances;
+    ProxyAffordanceMap.AffordanceDefinition[] m_AffordanceDefinitions;
```

Comment:
ProxyAffordanceMap is redundant with your alias above. Also see my comment about promoting this to a first-level class.